packages/core/test/tools/fetch-logger.js

```diff
@@ -0,0 +1,207 @@
+/* global Request */
+
+const assert = require('node:assert');
```

Comment:
👁️ Very nice, I've been interested in this built-in library! Should we go ahead and default to the `strict` version? `require('node:assert/strict')`

Reply:
Switched to `node:assert/strict` in [cb6bdd9](https://github.com/zapier/zapier-platform/pull/908/commits/cb6bdd9adc1d05f441254a35aec771e8cd14cf26). I actually caught  an issue with the non-strict `assert.equal()` where it sees `'foo'` and `['foo']` as equal. Using `node:assert/strict` and `assert.deepEqual()` fixed that, so thanks!